sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Customizations/ExecuteWranglingDataflowActivity.cs

```diff
@@ -0,0 +1,56 @@
+        /// class.
+        /// </summary>
+        /// <param name="name">Activity name.</param>
+        /// <param name="dataFlow">Data flow reference.</param>
+        /// <param name="additionalProperties">Unmatched properties from the
+        /// message are deserialized this collection</param>
+        /// <param name="description">Activity description.</param>
+        /// <param name="dependsOn">Activity depends on condition.</param>
+        /// <param name="userProperties">Activity user properties.</param>
+        /// <param name="staging">Staging info for execute data flow
+        /// activity.</param>
+        /// <param name="integrationRuntime">The integration runtime
+        /// reference.</param>
+        /// <param name="compute">Compute properties for data flow
+        /// activity.</param>
+        /// <param name="traceLevel">Trace level setting used for data flow
+        /// monitoring output. Supported values are: 'coarse', 'fine', and
+        /// 'none'. Type: string (or Expression with resultType string)</param>
+        /// <param name="continueOnError">Continue on error setting used for
+        /// data flow execution. Enables processing to continue if a sink
+        /// fails. Type: boolean (or Expression with resultType
+        /// boolean)</param>
+        /// <param name="runConcurrently">Concurrent run setting used for data
+        /// flow execution. Allows sinks with the same save order to be
+        /// processed concurrently. Type: boolean (or Expression with
+        /// resultType boolean)</param>
+        /// <param name="sinks">(Deprecated. Please use Queries). List of Power
+        /// Query activity sinks mapped to a queryName.</param>
+        /// <param name="queries">List of mapping for Power Query mashup query
+        /// to sink dataset(s).</param>
+        /// <param name="policy">Activity policy.</param>
+        public ExecuteWranglingDataflowActivity(string name, DataFlowReference dataFlow, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), string description = default(string), IList<ActivityDependency> dependsOn = default(IList<ActivityDependency>), IList<UserProperty> userProperties = default(IList<UserProperty>), DataFlowStagingInfo staging = default(DataFlowStagingInfo), IntegrationRuntimeReference integrationRuntime = default(IntegrationRuntimeReference), ExecuteDataFlowActivityTypePropertiesCompute compute = default(ExecuteDataFlowActivityTypePropertiesCompute), object traceLevel = default(object), object continueOnError = default(object), object runConcurrently = default(object), IDictionary<string, PowerQuerySink> sinks = default(IDictionary<string, PowerQuerySink>), IList<PowerQuerySinkMapping> queries = default(IList<PowerQuerySinkMapping>), ActivityPolicy policy = default(ActivityPolicy))
+            : base(name, additionalProperties, description, dependsOn, userProperties)
```

Comment:
Please make all the parameters before sinks ( included) as required. You can refer to https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/182581/Creating-Customization-Constructor